core/src/com/unciv/logic/battle/Battle.kt

```diff
@@ -77,7 +77,7 @@ object Battle {
         // Withdraw from melee ability
         if (attacker is MapUnitCombatant && attacker.isMelee() && defender is MapUnitCombatant) {
             val withdraw = defender.unit.getMatchingUniques(UniqueType.MayWithdraw)
-                .sumOf{ it.params[0].toInt() }  // If a mod allows multiple withdraw properties, ensure the best is used
```

Comment:
That was done on purpose, [see here](https://github.com/yairm210/Unciv/commit/644c61a85196842fae5d805e280189cd68968a3b) - only the comment was left from the old maxOfOrNull, no longer matching reality

Reply:
In that case it's still a bit weird; if a caravel has an 80% chance to withdraw and it is from a nation with a 40% chance to withdraw, should it have a 120% chance to withdraw? To me this didn't make sense, so I reverted it. But I suppose that in that case you could say it should have a `1-(1-80%)*(1-40%)` chance to withdraw. I'll chance the code accordingly